account_payment_order/views/account_payment_mode.xml

```diff
@@ -1,75 +1,71 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-
-
-<record id="account_payment_mode_form" model="ir.ui.view">
-    <field name="name">account_payment_order.account.payment.mode.form</field>
-    <field name="model">account.payment.mode</field>
-    <field name="inherit_id" ref="account_payment_mode.account_payment_mode_form"/>
-    <field name="arch" type="xml">
-        <field name="payment_type" position="after">
-            <field name="payment_order_ok"/>
-        </field>
-        <group name="main" position="after">
-            <group name="payment_order_options"
-                string="Options for Payment Orders"
-                attrs="{'invisible': [('payment_order_ok', '=', False)]}">
-                <field name="no_debit_before_maturity"
-                    attrs="{'invisible': ['|', ('payment_type', '!=', 'inbound'), ('payment_order_ok', '!=', True)]}"/>
-                <field name="default_date_prefered"/>
-                <field name="group_lines"/>
-            </group>
-            <group name="payment_order_create_defaults"
-                    string="Select Move Lines to Pay - Default Values"
-                    attrs="{'invisible': [('payment_order_ok', '=', False)]}">
-                <field name="default_journal_ids" widget="many2many_tags"/>
-                <field name="default_payment_mode"/>
-                <field name="default_target_move" widget="radio"/>
-                <field name="default_invoice"/>
-                <field name="default_date_type"/>
-            </group>
-            <group name="accounting-config"
-                    string="Accounting Entries Options"
-                    attrs="{'invisible': [('payment_order_ok', '=', False)]}">
-                <field name="generate_move"/>
-                <field name="offsetting_account" widget="radio"
-                    attrs="{'required': [('generate_move', '=', True)], 'invisible': [('generate_move', '=', False)]}"/>
-                <field name="transfer_account_id"
-                    attrs="{'invisible': [('offsetting_account', '!=', 'transfer_account')], 'required': [('offsetting_account', '=', 'transfer_account')]}"
-                    context="{'default_reconcile': True, 'default_company_id': company_id}"/> <!-- We can't put a default vue to user_type_id... -->
-                <field name="transfer_journal_id"
-                    attrs="{'invisible': [('offsetting_account', '!=', 'transfer_account')], 'required': [('offsetting_account', '=', 'transfer_account')]}"/>
-                <field name="move_option"
-                    attrs="{'invisible': [('generate_move', '=', False)], 'required': [('generate_move', '=', True)]}"/>
-                <field name="post_move"/>
+    <record id="account_payment_mode_form" model="ir.ui.view">
```

Comment:
All these indentation changes that can be considered refactoring (and although is good that you want to have it), please include them in a separate commit, not in the migration commit, as they add a lot of noise here and they are not exclusive of the migration.